tox.ini

```diff
@@ -0,0 +1,53 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported Python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py27,py35,flake8
+skip_missing_interpreters = False
+
+[travis]
+python =
+  3.5: py35, flake8
+
+[flake8]
+max-line-length = 120
+exclude =
+    build/
+    .git
+    __pycache__
+    .tox
+    tests/data/
+    venv/
+max-complexity = 15
+
+[testenv]
+# TEAMCITY_VERSION environment variable exists during build on Teamcity. teamcity-messages uses it in order to enable
+# reporting to TeamCity.
+passenv =
+    TEAMCITY_VERSION
+    AWS_ACCESS_KEY_ID
+    AWS_SECRET_ACCESS_KEY
+    AWS_SESSION_TOKEN
+# {posargs} can be passed in by additional arguments specified when invoking tox.
+# Can be used to specify which tests to run, e.g.: tox -- -s
+commands =
+    coverage run --source sagemaker_containers -m py.test {posargs}
+    {env:IGNORE_COVERAGE:} coverage report --fail-under=90
+deps =
+    pytest
+    pytest-cov
+    pytest-xdist
+    mock
+    contextlib2
+    teamcity-messages
+    awslogs
+
+[testenv:flake8]
+basepython = python
+deps =
+    flake8
+    pep8-naming
+    flake8-import-order
+commands = flake8
```

Comment:
don't forget a newline at the end of the file

Reply:
done